Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
 ```
 
 #### Copy the data directory of the now-idle member to the new machine
 
+`%data_dir%` is usually `/var/lib/etcd2`.
+
 ```
-$ tar -cvzf infra1.etcd.tar.gz %data_dir%
+$ cd %data_dir%
+$ tar -cvzf infra1.etcd.tar.gz *
```

Comment:
Replace `tar -xzvf infra1.etcd.tar.gz -C %data_dir%` with `tar -xzvf infra1.etcd.tar.gz -C /`? Then it should work for all cases